_docs/instructor/autograding/jupyter_notebook.md

```diff
@@ -0,0 +1,247 @@
+  "container_options": {
+    "container_image": "submitty/jupyter:latest"
+  },
+  "allow_system_calls": [
+    "COMMUNICATIONS_AND_NETWORKING_INTERPROCESS_COMMUNICATION",
+    "COMMUNICATIONS_AND_NETWORKING_KILL",
+    "COMMUNICATIONS_AND_NETWORKING_SIGNALS",
+    "COMMUNICATIONS_AND_NETWORKING_SOCKETS",
+    "COMMUNICATIONS_AND_NETWORKING_SOCKETS_MINIMAL",
+    "FILE_MANAGEMENT_MOVE_DELETE_RENAME_FILE_DIRECTORY",
+    "FILE_MANAGEMENT_RARE",
+    "PROCESS_CONTROL_ADVANCED",
+    "PROCESS_CONTROL_NEW_PROCESS_THREAD",
+    "PROCESS_CONTROL_MEMORY_ADVANCED",
+    "PROCESS_CONTROL_SYNCHRONIZATION",
+    "PROCESS_CONTROL_SCHEDULING",
+    "FILE_MANAGEMENT_PERMISSIONS",
+    "UNKNOWN"
+  ],
+  "testcases": [
+    {
+      "title": "Executes successfully",
+      // Although wildcard is used, the grader will only expect one notebook file. Use if filename cannot be determined.
+      "command": "jupyter_notebook_grader -i *.ipynb -o executed.ipynb",
+      "points": 1,
+      "validation": [
+        {
+          "method": "fileExists",
+          "actual_file": "executed.ipynb",
+          "show_actual" : "always",
+          "show_message" : "always"
+        }
+      ]
+    },
+    {
+      "pre_commands" : [
+        {
+          "command" : "cp",
+          "testcase" : "test01",
+          "source" : "cell1*.*",
+          "destination" : "./"
+        }
+      ], 
+      "title": "STDOUT",
+      "points": 1,
+      "validation": [
+        {
+          "method": "diff",
+          "actual_file": "cell1_stdout.txt",
+          "expected_string" : "hello world!"
+        },
+        {
+          "method" : "warnIfEmpty",
+		      "actual_file" : "cell1_stderr.txt",
+		      "show_actual" : "always",
+          "deduction" : 0.0
+        },
+        {
+          "method" : "warnIfEmpty",
+		      "actual_file" : "cell1_source.txt",
+		      "show_actual" : "always",
+          "deduction" : 0.0
+        },
+        {
+          "method" : "errorIfNotEmpty",
+		      "actual_file" : "cell1.err",
+		      "show_actual" : "always",
+          "deduction" : 0.0
+        }
+      ]
+    },
+    {
+      "pre_commands" : [
+        {
+          "command" : "cp",
+          "testcase" : "test01",
+          "source" : "cell2*.*",
+          "destination" : "./"
+        }
+      ], 
+      "title": "Markdown",
+      "points": 1,
+      "validation": [
+        {
+          "method": "diff",
+          "actual_file": "cell2.txt",
+          "expected_string" : "## 2. Create IPython images with the given \"one.png\" and \"two.png\".\nUse `from IPython.display import Image`"
+        },
+        {
+          "method" : "errorIfNotEmpty",
+		      "actual_file" : "cell2.err",
+		      "show_actual" : "always",
+          "deduction" : 0.0
+        }
+      ]
+    }
+  ]
+}
+```
+
+### Required Fields
+
+For autograding Jupyter Notebooks, the following fields must be included in your autograding configuration.
+
+```json
```

Comment:
again, please don't include the full details of the values here, but you can call out the field names that are important to override from the defaults to make a typical jupyter notebook gradeable work.